dev/Deployment/DeploymentManager.cpp

```diff
@@ -327,7 +327,11 @@ namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::implem
             attributeCount++;
         }
 
-        THROW_IF_WIN32_BOOL_FALSE(InitializeProcThreadAttributeList(nullptr, attributeCount, 0, &attributeListSize));
+        // attributeCount is always >0 so we need to allocate a buffer. Call InitializeProcThreadAttributeList()
+        // to determine the size needed so we always expect ERROR_INSUFFICIENT_BUFFER.
+        THROW_HR_IF(E_UNEXPECTED, !!InitializeProcThreadAttributeList(nullptr, attributeCount, 0, &attributeListSize));
+        const auto lastError{ GetLastError() };
+        THROW_HR_IF(HRESULT_FROM_WIN32(lastError), lastError != ERROR_INSUFFICIENT_BUFFER);
         PPROC_THREAD_ATTRIBUTE_LIST attributeList = reinterpret_cast<PPROC_THREAD_ATTRIBUTE_LIST>(new BYTE[attributeListSize]);
```

Comment:
As Alex mentioned earlier, it looks like this is never released. It's a small, fixed, one time leak, so lower priority, but still good to either address or track.